test/utils.py

```diff
@@ -79,3 +80,10 @@ def wait_file(dir_name, file_pattern, exclude_list, timeout_sec=1):
         iter_count = iter_count + 1
 
     return ""
+
+
+def kill_instance(instance_file_name):
```

Comment:
Why not pid number?
We may want to run tests simultaneously in two different branches, for example

Reply:
Done. Now it uses kill by PID